### ISSUE TITLE
[Bug]: Changing Guard Mode doesn't work with SoloCam S220

### ISSUE THREAD

**Issue by gintonic4all** [homebridge-eufy-security/plugin#386]
### What happened?

Got a new Eufy S220 SoloCam. 
It's not possible to configure it as camera device for the Home-App. And it's motion sensor doesn't appear in the Home-App.

Changing Guard-Mode lead to restart of homebridge (but only once). 
And somehow return values cannot be interpreted. (see log information below)

Maybe it's due to the new cam model, which is not supported by this homebridge-plugin, yet.

Best regards, 
GinTonic4All  

### Device Type

other (UnSupported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.6.x (Supported)

### NodeJS Version

v18 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
[08/29/2023 18:04:05]   [EufySecurity-2.2.5]    INFO    [StationAccessory.ts:335]       T8134P202323563D Request to change station guard mode to: Away.
[29/08/2023, 20:04:07] NotSupportedError: This functionality is not implemented or supported
    at Station.setGuardMode (/usr/local/lib/node_modules/homebridge-eufy-security/node_modules/eufy-security-client/src/http/station.ts:688:19)
    at StationAccessory.handleSecuritySystemTargetStateSet (/usr/local/lib/node_modules/homebridge-eufy-security/src/plugin/accessories/StationAccessory.ts:336:24)
    at SecuritySystemTargetState.<anonymous> (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:2547:42)
    at step (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:195:27)
    at Object.next (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:176:57)
    at /usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:169:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/usr/local/lib/node_modules/homebridge/node_modules/tslib/tslib.js:165:16)
    at SecuritySystemTargetState.Characteristic.handleSetRequest (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/dist/lib/Characteristic.js:871:24)
    at Bridge.<anonymous> (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1894:29)
[29/08/2023, 20:04:07] Got SIGTERM, shutting down Homebridge...


[08/29/2023 18:04:20]   [EufySecurity-2.2.5]    WARN    [StationAccessory.ts:339]       T8134P202323563D Changing guard mode to Awaydid not complete. Retry...
[29/08/2023, 20:04:20] NotSupportedError: This functionality is not implemented or supported
    at Station.setGuardMode (/usr/local/lib/node_modules/homebridge-eufy-security/node_modules/eufy-security-client/src/http/station.ts:688:19)
    at Timeout._onTimeout (/usr/local/lib/node_modules/homebridge-eufy-security/src/plugin/accessories/StationAccessory.ts:342:26)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7)
[29/08/2023, 20:04:20] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null
[29/08/2023, 20:04:25] [HB Supervisor] Restarting Homebridge...


[08/29/2023 18:15:41]   [EufySecurity-2.2.5]    ERROR   [StationAccessory.ts:301]       T8134P202323563D handleSecuritySystemCurrentStateGet Wrong return value
```

### config

```shell
... "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        },
        {
            "username": "xyz@abc.com",
            "password": "zyx",
            "country": "DE",
            "deviceName": "",
            "platform": "EufySecurity",
            "ignoreStations": [],
            "ignoreDevices": [],
            "cleanCache": true
        },...
```

**Comment by lenoxys** [comment]
**Thank you for taking the time to report this issue.**

As part of a repository cleanup, we are closing issues that have been open for more than 12 months without resolution. A **beta version** of the plugin has been released that includes significant changes and improvements.

We encourage you to try the latest beta and see if your issue has been addressed. If the problem persists, please feel free to open a new issue with updated details and logs from the latest version.

Thank you for your patience and support!
